charts/actions-runner-controller/README.md

```diff
@@ -72,6 +72,7 @@ All additional docs are kept in the `docs/` folder, this README is solely for do
 | `scope.watchNamespace`                                   | Tells the controller and the github webhook server which namespace to watch if `scope.singleNamespace` is true             | `Release.Namespace` (the default namespace of the helm chart).       |
 | `scope.singleNamespace`                                  | Limit the controller to watch a single namespace                                                                           | false                                                                |
 | `certManagerEnabled`                                     | Enable cert-manager. If disabled you must set admissionWebHooks.caBundle and create TLS secrets manually                   | true                                                                 |
+| `useCustomRunnersRBAC`                                   | Use custom RBAC for runners (role, role binding and service account), this will enable reporting runner statuses           | false                                                                |
```

Comment:
```suggestion
| `useRunnerStatusUpdateHookEphemeralRole `                                   | Use custom RBAC for runners (role, role binding and service account), this will enable reporting runner statuses           | false                                                                |
```